maxim/logger/gemini/client.py

```
@@ -158,22 +186,51 @@ def send_message_stream(
+        """Set an attribute on the chat session.
+
+        Args:
+            name: The name of the attribute.
+            value: The value of the attribute.
+        """
         if name in ("_chat", "_logger", "_trace_id", "_is_local_trace"):
             super().__setattr__(name, value)
         else:
             setattr(self._chat, name, value)
 
     def end_trace(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `end_trace`

Add return type annotation: `None`

(ANN201)